### PR TITLE
Remove loader

### PR DESCRIPTION
**Pull request by philandstuff**
[The loader now has its own repo](https://github.com/openregister/loader)!

So we can safely remove it from this repo.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,20 +1,72 @@
...
 
-apply from: 'idea-config.gradle'
+assemble {
+    doLast {
+        new File("deploy/mint.jar").bytes = new File("build/libs/mint.jar").bytes
+        createDeployableBundle.execute()
+    }
+}
 
-task wrapper(type: Wrapper) {
```
Comment (om-sharma) [inline build.gradle:17]: I think we still need wrapper task. It is required to regenerate contents of gradle directory when update the gradle version.

Comment (philandstuff) [inline build.gradle:17]: it's still there, just at line 6 now

**Comment by om-sharma** [comment]
I think we can now remove the settings.gradle file also. it was required to specify submodule of projects.

File: build.gradle (hunk 1 above)
Comment (om-sharma) [inline build.gradle:17]: wait! I can see its there at the top in the file.

**Pull request merged**